### ISSUE TITLE
Support scripting

### ISSUE THREAD

**Issue by Uxio0** [gnosis/safe-cli#36]
Currently the only way to use the `safe-cli` is using the interactive mode. For example, to send some ether a user must run the app, load the owners and then call `send_ether` and press intro.

An automatic way to do it, like `python safe_cli --node <node_url> --private-key <owner_private_key_1> --private-key <owner_private_key_2> send-ether --wei <ether_in_wei> --to <address>`

**Comment by ademidun** [comment]
Agreed. Especially, because the `interactive` mode requirement also makes it unusable in remote servers like Google Colab. I made a [Jupyter notebook](https://colab.research.google.com/gist/ademidun/003eee3ccdd3daa21943846c05f68b18/safe_cli.ipynb) to make it easy for users to run the safe-cli from the browser since some users won't be able to install the command line themselves.

I get the following error: 
```
WARNING: your terminal doesn't support cursor position requests (CPR).
```

Which I think is related to the lack of interactivity. So this causes the load safe command to hang and never complete.

I think this is a high priority fix as [many](https://ethereum.stackexchange.com/questions/144174/how-to-recover-tokens-send-to-ethereum-instead-of-polygon-1-3-0/144186#144186) [issues](https://ethereum.stackexchange.com/questions/123289/copy-gnosis-safe-on-ethereum-to-safe-on-polygon-with-same-safe-address/144187#144187) that users face through UI can be solved through CLI and many users may not be able to use the CLI but they can use the notebook.

**Comment by Uxio0** [comment]
Take a look at https://typer.tiangolo.com/

**Comment by JagoFigueroa** [comment]
Tested and verified, many thanks!
